exception.py

```diff
@@ -0,0 +1,37 @@
+"""
+Define your custom class for exceptions. Show an example how it might be used.
+"""
+
+
+class Error(Exception):
+    pass
+
+
+class OtherTypeError(Error):
+    pass
+
+
+class DivideByZeroError(Error):
+    pass
+
+
+def zero(a, b):
+    try:
+        if isinstance(a, str) or isinstance(b, str):
+            raise OtherTypeError('Invalid type')
+        elif int(b) == 0:
+            raise DivideByZeroError('Division by zero')
+        else:
+            print("{} / {} = {}".format(a, b, (a / b)))
+    except OtherTypeError as other_type:
```

Comment:
it's better to name it as `other_type_error`, etc. to make sure it's clear what is the content of this variable.

Reply:
done